docker/dev/Dockerfile

```diff
@@ -77,6 +47,5 @@ ENV DATABASE=sqlite \
     TEST_WORKSPACE=/CodeCompass/test_workspace \
     TEST_DB="sqlite:database=$TEST_WORKSPACE/cc_test.sqlite"
 
-ENV PATH="$INSTALL_DIR/bin:/opt/thrift_install/bin:/opt/odb_install/bin:$PATH"
-ENV CMAKE_PREFIX_PATH="/opt/thrift_install:/opt/odb_install:$CMAKE_PREFIX_PATH"
+ENV PATH="$INSTALL_DIR/bin:$PATH"
 ENV LD_LIBRARY_PATH="$INSTALL_DIR/lib:$LD_LIBRARY_PATH"
```

Comment:
I think this line is not needed, because "installdir/lib" is automatically set as search root by the server, and plugins are built with RPATH.

https://github.com/Ericsson/CodeCompass/blob/3f413ed284ced641a00de64a8b9ef41232b8515f/webserver/src/webserver.cpp#L51

https://github.com/Ericsson/CodeCompass/blob/3f413ed284ced641a00de64a8b9ef41232b8515f/Config.cmake#L48-L61